dusty/commands/utils.py

```diff
@@ -1,8 +1,16 @@
+from copy import copy
 import os
 import subprocess
 
+from ..systems.docker import get_docker_env
+
+
 def _executable_path(executable_name):
     return subprocess.check_output(['which', executable_name]).strip()
 
+
```

Comment:
Ok. My linter complains at me for only having one, but I suppose it's better to be consistent throughout the project